modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -0,0 +1,120 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-modify-machine-set-aws_{context}"]
+
+= Adding an ARM64 compute machine set to your cluster 
+
+To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster and creates a cluster with multi-architecture compute machines.
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`). 
+* You created an AMD64 single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary.
+
+
+.Procedure
+
+* Create and modify a compute machine set, this will control the ARM64 compute nodes in your cluster. 
```

Comment:
Based on other compute machine set procedures, this feels like it's missing a step or two. Usually:

1.  Create a YAML file that contains the compute machine set custom resource (CR) sample and is named `<file_name>.yaml`.
2. Fiddle with the values. The sample YAML is helpful here, and they can also look at one of the default compute machine sets on the cluster to see cluster-specific values they might need to keep the same.
3. Create the new `MachineSet` CR with `oc create -f <file_name>.yaml`

Reply:
Oh gotcha! Yeah lemme look at those examples